project/BuildPlugin.scala

```diff
@@ -533,12 +533,16 @@ object BuildImplementation {
       sbtbuildinfo.BuildInfoPlugin.buildInfoScopedSettings(Test) ++ List(
         Keys.fork in Test := true,
         Keys.resolvers ++= List(
-          MavenRepository("Gradle releases", "https://repo.gradle.org/gradle/libs-releases-local/")
+          MavenRepository("Gradle releases", "https://repo.gradle.org/gradle/libs-releases-local/"),
+          MavenRepository("Android plugin", "https://maven.google.com/"),
+          MavenRepository("trove4j dependency", "https://jcenter.bintray.com/"),
```

Comment:
I don't think `bintray` is working still, so this might not work. Also looks like there is nothing about `trove4j` in the code. Maybe leftover from previous approach?

Reply:
Trove4J is a transitive dependency required by the android plugin but it's now on Maven Central so I've removed